src/features/enrollments/components/StudentEnrollmentsPage/index.jsx

```diff
@@ -149,21 +220,12 @@ const StudentEnrollmentsPage = () => {
         onPageSelect={handlePagination}
       />
       <AlertModal
```

Comment:
Review the UX for the modal because I see the scrollbar when is not neccesary
![Screenshot from 2025-06-05 13-44-45](https://github.com/user-attachments/assets/fa0b1c0b-4add-4bfe-9da6-11fd5c5a26ab)